services/multilang-support/Dockerfile

```diff
@@ -20,4 +20,4 @@ COPY services/multilang-support/src/*.py /app/src/
 
 EXPOSE 80
 ENV FLASK_APP=src/app.py
-CMD [ "gunicorn", "src.translate:app", "-b", "0.0.0.0:80", "--log-level=debug"]
+CMD [ "gunicorn", "src.translate:app", "--bind", "0.0.0.0:80", "--log-level=info", "--timeout=120", "--workers=2"]
```

Comment:
Notes on changes here:
- `--log-level=info`: Python `logging` still allows debug level message to passthrough and printed on the terminal/console of the docker container.
- `--timeout=120`: this helps prevent worker timeout error that occurs when starting, workers take extra time to download models and restarts after one or two models are instantiated, which lead to extremely long boot-up time for the Translator if we support more languages.
- `--workers=2`: a nice feature to have, help processing multiple requests concurrently.

Reply:
UPDATE: removed `--workers` flag due to some unexplicable behaviour between them while running the service :)

Reply:
Thanks on removing workers, I'd honestly rather handle scaling up at the docker level rather than hard coded into the image itself!